data_summaries/Cobra Frames.json

```diff
@@ -0,0 +1,54 @@
+{
+    "CobraFrames-gpt3.5": {
+        "Unique Dataset Identifier": "CobraFrames-gpt3.5",
+        "Dataset Name": "Cobra Frames-gpt3.5",
+        "Paper Title": "COBRA Frames: Contextual Reasoning about Effects and Harms of Offensive Statements",
+        "GitHub URL": "https://github.com/XuhuiZhou/cobra-frames",
+        "Dataset URL": "https://huggingface.co/datasets/cmu-lti/cobracorpus/blob/main/toxigen_explanations.csv",
+        "Hugging Face URL": "https://huggingface.co/datasets/cmu-lti/cobracorpus/blob/main/toxigen_explanations.csv",
+        "Papers with Code URL": "",
+        "ArXiv URL": "https://arxiv.org/abs/2306.01985",
+        "Semantic Scholar Corpus ID": "259075355",
+        "Collection": "Cobra Frames",
+        "Collection URL": "https://cobra.xuhuiz.com",
+        "Languages": [
+            "English"
+        ],
+        "Task Categories": [
+            "Bias Detection",
+            "Binary Classification of Hate Speech",
+            "Binary Classification of Racial Bias",
+            "Offensive Language Detection",
+            "Toxicity Detection",
+            "Sarcasm Detection",
+            "Stereotype Detection"
+        ],
+        "Text Sources": [
+            "crowdsourced",
+            "text-davinci-003"
+        ],
+        "Model Generated": ["OpenAI GPT-3"],
+        "Format": [
+            "Zero-shot",
+            "Few-shot"
+        ],
+        "Human Annotation": "Yes",
+        "Derived from Datasets": [
+            "ToxiGen"],
+        "Creators": [
+            "Carnegie Mellon University",
+            "Allen Institute for AI"
+        ],
+        "Licenses": [
+            {
+                "License": "BigScience OpenRAIL-M",
+                "License URL": "https://huggingface.co/blog/open_rail"
+            }
+        ],
```

Comment:
Add the OpenAI license as well, since it used GPT-3 to generate as well.

Reply:
Added, not sure about the url tho